server.js

```diff
@@ -0,0 +1,31 @@
+const express = require("express");
+const webpack = require("webpack");
+const path = require("path");
+const open = require("open");
+const dotenv = require("dotenv");
+const config = require("./webpack.config");
+
+/* eslint-disable no-console */
+
+dotenv.config();
+
+const port = process.env.PORT || 8989;
+const app = express();
+const compiler = webpack(config);
+
+app.use(require("webpack-dev-middleware")(compiler, {
+  noInfo: true,
+  publicPath: config.output.publicPath
+}));
+
+app.use(require("webpack-hot-middleware")(compiler));
+
+app.get("*", function(req, res) {
+  res.sendFile(path.join( __dirname, "./client/index.html"));
+});
+
+app.listen(port, err =>{
+  if (!err) open(`http://localhost:${port}`);  else {
```

Comment:
## Nitpick

Have you heard of `tenary` operators, if you can make use of it, then it'll be great. If not, you can just skip it and add it to the list of things we'll talk about during our next session.